src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -6,110 +6,110 @@
 import { CancellationTokenSource } from 'vs/base/common/cancellation';
 import * as errors from 'vs/base/common/errors';
 import { Emitter, Event } from 'vs/base/common/event';
+import { combinedDisposable } from 'vs/base/common/lifecycle';
```

Comment:
a little scared of this huge import change in such a crucial file tbh

Reply:
For this, I'll revert import changes and just have the added registery.